index.html

```diff
@@ -33,11 +33,15 @@ <h3>Welcome</h3>
 <p>The Corosync Cluster Engine is a Group Communication System with additional features for implementing high availability within applications. The project provides four C Application Programming Interface features:</p>
 
 <ul>
-<li>A closed process group communication model with virtual synchrony guarantees for creating replicated state machines.</li>
+<li>A closed process group communication model with some guarantees derived from extended virtual synchrony for creating replicated state machines.</li>
```

Comment:
why "some" guarantees? Just add "extended" is enough.

Reply:
And that's what I am talking about, that application is going to be
written by someone who is likely evaluating corosync starting with
reviewing this very page, so I don't share the opinion it's OK to
make overly generalized claims (especially towards someone with some
theoretical backgrounds, which is likely) and pretending
the full blown (complete) implementation is offered, since you
confirmed it's not true.

Alternatively, it may be worder explicitly:

```
with extended synchrony guarantees (excluding its strongest notion
for the time being)
```

or something along those lines.  That would be a fair claim towards
potential client SW authors (as that might help to prevent possible
subtle lapses arising from inappropriate use of this functionality).